_data/projects/ISOI.yml

```
@@ -0,0 +1,12 @@
+name: Instrument Society Of India
+desc: An Open Source Repository for Hacktoberfest 22. Looking for contributor who look forward to actively participate in this Hacktoberfest
+site: https://github.com/Instrumentation-Society-Of-India-VIT/HacktoberFest22/
+tags:
+  - arduino
+  - javascript
+  - blockchain
+  - hardware
+  - iot
+upforgrabs:
+  name: help wanted
+  link: https://github.com/Instrumentation-Society-Of-India-VIT/HacktoberFest22/
```

🤔 this link should point to a specific label, but this one does not have any issues currently

```suggestion
  link: https://github.com/Instrumentation-Society-Of-India-VIT/HacktoberFest22/labels/help%20wanted
```

This means the project won't be listed on the site as there's no tasks available to work on. Do you think this will be populated as part of Hacktoberfest @YUVRAJ06singh08deora ?